apps/web/src/pages/templates/components/VariantItemCard.tsx

```diff
@@ -127,6 +132,9 @@ export const VariantItemCard = ({
   variant: IFormStep | IVariantStep;
   nodeType: NodeType;
   'data-test-id'?: string;
+  errorMessage?: string;
+  isActiveError?: boolean;
```

Comment:
isActiveError is used to sync with the error in the navigation and scroll that particular variantNode to view and to also add the error border around it